app/src/main/java/com/kickstarter/libs/Either.kt

```diff
@@ -0,0 +1,61 @@
+package com.kickstarter.libs
+
+sealed class Either<out A, out B> {
+  class Left<out L, out R>(internal val left: L) : Either<L, R>()
```

Comment:
I'd use **Nothing** as the unsued type in Left and Right,
e.g. _class Left<out L>(internal val left: L) : Either<L, Nothing>()_

This allows to create an instance of Left with _val left = Left(15)_ without specifying the right type.